### PR TITLE
Add LlamaCoin

### PR DESCRIPTION
**Pull request by DennisD076**

### CODE REVIEW

File: tokens/main/0x8cE14e9906C64F8e17fA27eb51d3Db1df3DA2c16/additional.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (libotony) [inline tokens/main/0x8cE14e9906C64F8e17fA27eb51d3Db1df3DA2c16/additional.json:1]: Folder should be the address in lower case, please update accordingly.

**Review by libotony — CHANGES_REQUESTED** [review]

**Comment by DennisD076** [comment]
address is added in lower case. Thanks!

**Comment by libotony** [comment]
@DennisD076  Image should be 256px * 256px

**Review by libotony — APPROVED** [review]

**Comment by libotony** [comment]
Info from script, name does not match contract name (info=LlamaCoin, contract=LLAMACOIN).

**Comment by libotony** [comment]
Info from script, symbol does not match contract symbol (info=LLAMA, contract=LLAMACOIN).

**Comment by libotony** [comment]
> Info from script, symbol does not match contract symbol (info=LLAMA, contract=LLAMACOIN).

I think `LLAMA` is the right symbol, correct? @DennisD076, But it seems to be not correctly configured at the contract level.

**Comment by DennisD076** [comment]
I deployed the SC with LLAMACOIN symbol aswell, keep it that way please

**Comment by DennisD076** [comment]
Not adjustable anymore

**Pull request merged**